src/main/java/com/github/pyenvpipeline/jenkins/VirtualenvManager.java

```diff
@@ -98,12 +98,40 @@ public ArgumentListBuilder getCreateVirtualEnvCommand(StepContext context, Strin
         LOGGER.info("Creating virtualenv at " + fullQualifiedDirectoryName + " using Python installation " +
                 "found at " + commandPath);
 
-        ArgumentListBuilder command = new ArgumentListBuilder();
+        // Checking version of python
+        ArgumentListBuilder versionChecker = new ArgumentListBuilder();
+        versionChecker.add(commandPath);
+        versionChecker.add("--version");
+        String versionOutput = runCommandList(versionChecker, context);
+
+        String[] outputPortions = versionOutput.split(Pattern.quote(" "));
+        String versionPortion = outputPortions[outputPortions.length-1];
+        String[] versionsPortions = versionPortion.split(Pattern.quote("."));
+
+        boolean prePEP405 = true;
+
+        if (versionsPortions.length >= 3) {
+            Integer major = Integer.parseInt(versionsPortions[0]);
+            Integer minor = Integer.parseInt(versionsPortions[1]);
+
+            if ((major > 3) || ((major == 3) && (minor >= 7))) {
```

Comment:
PEP405 was released with python 3.3, might be worth dropping this to minor >= 3

Reply:
For the purposes of this pull requests, I am fine with simply supporting Python 3.7. I agree that we should look into dropping the minor version further; however, that would require more extensive testing, and I would like to include this sooner than later since it appears to be a bit of a show-stopper for 3.7+